### PR TITLE
Update User Guide and Developers Guide

### PR DESCRIPTION
**Pull request by hiyingnn**

### CODE REVIEW

File: docs/UserGuide.adoc [hunk 1]
```diff
@@ -143,11 +149,13 @@ Notifies all members with at least a minimum amount of loyalty points and with t
 
 Format: `notifyEmail [l/LOYALTY_POINTS_MINIMUM] [se/TRUE_FALSE]`
```
Comment (lingyanhao) [inline docs/UserGuide.adoc:150]: Are we still supporting this feature????
Didn't we agree to take this feature down?

File: docs/UserGuide.adoc [hunk 2]
```diff
@@ -159,9 +167,11 @@ Notifies all members with at least a minimum amount of loyalty points and with t
 
 Format: `notifySMS [l/LOYALTY_POINTS_MINIMUM] [ss/TRUE_FALSE]`
```
Comment (lingyanhao) [inline docs/UserGuide.adoc:168]: Same as above, do we want to keep this feature

File: docs/UserGuide.adoc [hunk 3]
```diff
@@ -196,26 +208,30 @@ View recipes of dishes served at the restaurant.
 
 Format: `listRecipes`
 
-For each recipe, the list of ingredients and its corresponding amount in units required for the standard serving will be displayed.
+****
+• For each recipe, the list of ingredients and its corresponding amount in units required for the standard serving will be displayed.
 Each recipe will be indexed.
+****
```
Comment (lingyanhao) [inline docs/UserGuide.adoc:214]: In my opinion, ingredients should come before recipe- users are probably confused

File: docs/UserGuide.adoc [hunk 4]
```diff
@@ -196,26 +208,30 @@ View recipes of dishes served at the restaurant.
...
+• For all ingredient inputs, if ingredient is new, message appears as a prompt for user to `addIngredient` first, before adding recipe.
 
-Each ingredient input should have the format `1/INGREDIENT_1&AMOUNT_IN_UNITS` where ingredient and amount in standardised units are separated by `&`.
+• Each ingredient input should have the format `1/INGREDIENT_1&AMOUNT_IN_UNITS` where ingredient and amount in standardised units are separated by `&`.
 
-`AMOUNT_IN_UNITS` is allowed to take on a decimal and is in the units specified in  listIngredients.
+• `AMOUNT_IN_UNITS` is allowed to take on a decimal and is in the units specified in  listIngredients.
 
-Number of ingredient inputs should correspond to `n/NUMBER_OF_INGREDIENTS`. If it does not, a message will appear to say incorrect number of inputs.
+• Number of ingredient inputs should correspond to `n/NUMBER_OF_INGREDIENTS`. If it does not, a message will appear to say incorrect number of inputs.
```
Comment (lingyanhao) [inline docs/UserGuide.adoc:229]: In my opinion, you don't really need to say "a message will appear to say incorrect number of inputs"- "an error message will be shown" is simpler and better language.

File: docs/UserGuide.adoc [hunk 5]
```diff
@@ -196,26 +208,30 @@ View recipes of dishes served at the restaurant.
...
 
-For all ingredient inputs, if ingredient is new, message appears as a prompt for user to `addIngredient` first, before adding recipe.
+****
+• For all ingredient inputs, if ingredient is new, message appears as a prompt for user to `addIngredient` first, before adding recipe.
 
-Each ingredient input should have the format `1/INGREDIENT_1&AMOUNT_IN_UNITS` where ingredient and amount in standardised units are separated by `&`.
+• Each ingredient input should have the format `1/INGREDIENT_1&AMOUNT_IN_UNITS` where ingredient and amount in standardised units are separated by `&`.
 
-`AMOUNT_IN_UNITS` is allowed to take on a decimal and is in the units specified in  listIngredients.
+• `AMOUNT_IN_UNITS` is allowed to take on a decimal and is in the units specified in  listIngredients.
```
Comment (lingyanhao) [inline docs/UserGuide.adoc:227]: Eh... your current units system is integer right??

File: docs/UserGuide.adoc [hunk 6]
```diff
@@ -289,10 +311,13 @@ Add new ingredient into inventory. Previously not existing in inventory before.
 
 Format: `addIngredient in/INGREDIENT_NAME u/INGREDIENT_UNIT [a/AMOUNT] [w/INGREDIENT_WARNING_AMOUNT]`
 
-If amount is not specified, amount is set to 0.
-Units are chosen based on the units that the stock was bought. Amount of ingredients for each dish should be converted by the user himself before `addRecipe` is called.
+****
+• If amount is not specified, amount is set to 0.
+
+• Units are chosen based on the units that the stock was bought. Amount of ingredients for each dish should be converted by the user himself before `addRecipe` is called.
```
Comment (lingyanhao) [inline docs/UserGuide.adoc:317]: "by the user himself" looks weird (language issue, less urgent probably)

Comment (lingyanhao) [reply docs/UserGuide.adoc:317]: also, is "AMOUNT" integer?

**Review by lingyanhao — APPROVED** [review]

File: docs/UserGuide.adoc (hunk 1 above)
Comment (hiyingnn) [reply docs/UserGuide.adoc:150]: we can put it as [for v2.0]

File: docs/UserGuide.adoc (hunk 5 above)
Comment (hiyingnn) [reply docs/UserGuide.adoc:227]: will be implemented in v1.2

**Pull request merged**